test/test_files/BBa_J23101_from_genbank_to_sbol3_direct.nt

```diff
@@ -1,14 +1,14 @@
 
-<https://testing.sbol3.genbank/BBa_J23101> <http://sbols.org/v3#description> "constitutive promoter family member" .
-<https://testing.sbol3.genbank/BBa_J23101> <http://sbols.org/v3#displayId> "BBa_J23101" .
-<https://testing.sbol3.genbank/BBa_J23101> <http://sbols.org/v3#hasNamespace> <https://testing.sbol3.genbank/> .
-<https://testing.sbol3.genbank/BBa_J23101> <http://sbols.org/v3#hasSequence> <https://testing.sbol3.genbank/BBa_J23101_sequence> .
-<https://testing.sbol3.genbank/BBa_J23101> <http://sbols.org/v3#role> <https://identifiers.org/SO:0000804> .
-<https://testing.sbol3.genbank/BBa_J23101> <http://sbols.org/v3#type> <https://identifiers.org/SBO:0000251> .
-<https://testing.sbol3.genbank/BBa_J23101> <http://sbols.org/v3#type> <https://identifiers.org/SO:0000987> .
-<https://testing.sbol3.genbank/BBa_J23101> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://sbols.org/v3#Component> .
-<https://testing.sbol3.genbank/BBa_J23101_sequence> <http://sbols.org/v3#displayId> "BBa_J23101_sequence" .
-<https://testing.sbol3.genbank/BBa_J23101_sequence> <http://sbols.org/v3#elements> "tttacagctagctcagtcctaggtattatgctagc" .
-<https://testing.sbol3.genbank/BBa_J23101_sequence> <http://sbols.org/v3#encoding> <https://identifiers.org/edam:format_1207> .
-<https://testing.sbol3.genbank/BBa_J23101_sequence> <http://sbols.org/v3#hasNamespace> <https://testing.sbol3.genbank/> .
-<https://testing.sbol3.genbank/BBa_J23101_sequence> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://sbols.org/v3#Sequence> .
+<https://test.sbol3.genbank/BBa_J23101> <http://sbols.org/v3#description> "constitutive promoter family member" .
```

Comment:
Is this change in test URI intentional?

Reply:
Oh, I had previously shortened it to make it visually easier to debug file diffs, I guess, we can let it be this way?